xml/admin_rbd.xml

```diff
@@ -1872,4 +1872,288 @@ protocol feature mismatch, my 2fffffffffff &lt; peer 4010ff8ffacffff missing 401
+     </step>
+     <step>
+       <para>
+         <literal>ceph-csi</literal> requires the cephx credentials for
+         communicating with the &ceph; cluster. Generate a <filename>csi-rbd-secret.yaml</filename>
+         file similar to the example below, using the newly-created
+         &kube; user ID and cephx key:
+       </para>
+<screen>&prompt.kubeuser;cat &lt;&lt;EOF &gt; csi-rbd-secret.yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: csi-rbd-secret
+  namespace: default
+stringData:
+  userID: kubernetes
+  userKey: AQD9o0Fd6hQRChAAt7fMaSZXduT3NWEqylNpmg==
+EOF</screen>
+     </step>
+     <step>
+       <para>
+         Once generated, store the new secret object in &kube;:
+       </para>
+<screen>&prompt.kubeuser;kubectl apply -f csi-rbd-secret.yaml</screen>
+     </step>
+     <step>
+       <para>
+         Create the required ServiceAccount and RBAC ClusterRole/ClusterRoleBinding
+         &kube; objects. These objects do not necessarily need to be customized
+         for your &kube; environment, and therefore can be used directly from the
+         <literal>ceph-csi</literal> deployment YAML files:
+       </para>
+<screen>&prompt.kubeuser;kubectl apply -f https://raw.githubusercontent.com/ceph/ceph-csi/master/deploy/rbd/kubernetes/csi-provisioner-rbac.yaml
+&prompt.kubeuser;kubectl apply -f https://raw.githubusercontent.com/ceph/ceph-csi/master/deploy/rbd/kubernetes/csi-nodeplugin-rbac.yaml</screen>
+     </step>
+     <step>
+       <para>
+         Create the <literal>ceph-csi</literal> provisioner and node plugins:
+       </para>
+<screen>&prompt.kubeuser;wget https://raw.githubusercontent.com/ceph/ceph-csi/master/deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml
+&prompt.kubeuser;kubectl apply -f csi-rbdplugin-provisioner.yaml
+&prompt.kubeuser;wget https://raw.githubusercontent.com/ceph/ceph-csi/master/deploy/rbd/kubernetes/csi-rbdplugin.yaml
+&prompt.kubeuser;kubectl apply -f csi-rbdplugin.yaml</screen>
+       <important>
+         <para>
+            By default, the provisioner and node plugin YAML files will pull the
+            development release of the <literal>ceph-csi</literal> container. The YAML files should
+            be updated to use a release version.
+         </para>
+       </important>
+     </step>
+   </procedure>
+   <sect2 xml:id="using-rbd-kubernetes">
+     <title>Using &ceph; Block Devices in &kube;</title>
+     <para>
+       The &kube; StorageClass defines a class of storage. Multiple StorageClass
+       objects can be created to map to different quality-of-service levels and
+       features. For example, NVMe versus HDD-based pools.
+     </para>
+     <para>
+       To create a <literal>ceph-csi</literal> StorageClass that maps to the
+       &kube; pool created above, the following YAML file can be used, after
+       ensuring that the <literal>clusterID</literal> property matches your
+       &ceph; cluster’s FSID:
```

Comment:
```suggestion
       &ceph; cluster's FSID:
```

Reply:
I think our policy is to use straight quotes:

'

Not curly quotes:

’

It's hard to see in some fonts.
’'’'’'’ ← in case it is clearer

Reply:
Probably a copy-pasta error from upstream :)